src/Amadeus/Client/Struct/Service/StandaloneCatalogue/FlightInfo.php

```diff
@@ -0,0 +1,124 @@
+<?php
```

Comment:
Same as above. No unnessary duplicate code.

Reply:
We need FlightInfo.php  here because of request parameter is different from one used in informative pricing request. We need <flightDetails> instead of <segmentInformation> used in InformativePricing Request.